index.html

```diff
@@ -716,389 +716,343 @@ <h2>Identifier</h2>
+  <a>DID</a> is deactivated, it is intended that it never be repurposed.
+        </p>
+  
+        <p>
+  Ideally, a <a>DID</a> would be a completely abstract decentralized identifier
+  (like a <a>UUID</a>) that could be bound to multiple underlying
+  <a>DID registries</a> over time, thus maintaining its persistence independent of
+  any particular system. However, registering the same identifier on multiple
+  <a>DID registries</a> introduces extremely hard entityship and
+  <a href="https://en.wikipedia.org/wiki/List_of_DNS_record_types%23SOA">start-of-authority</a>
+  (SOA) problems. It also greatly increases implementation complexity for
+  developers.
+        </p>
+  
+        <p>
+  To avoid these issues, it is RECOMMENDED that <a>DID method</a> specifications
+  only produce <a>DIDs</a> and <a>DID methods</a> bound to strong, stable
+  <a>DID registries</a> capable of making the highest level of commitment to
+  persistence of the <a>DID</a> and <a>DID method</a> over time.
+        </p>
+  
+        <p class="note">
+  Although not included in this version, future versions of this specification
+  might support a <a>DID document</a> <code>equivID</code> property to establish
+  verifiable equivalence relations between <a>DIDs</a> representing the same
+  subject on multiple <a>DID registries</a>. Such equivalence relations can
+  produce the practical equivalent of a single persistent abstract <a>DID</a>. For
+  more information, see Section <a href="#future-work"></a>.
+        </p>
+      </section>
+     </section>
 
     <section>
-      <h2>Normalization</h2>
+      <h3>DID URLs</h3>
 
-      <p>
-For the broadest interoperability, make <a>DID</a> normalization as simple and
-universal as possible:
-      </p>
-      <ul>
-        <li>
-The <a>DID scheme</a> name MUST be lowercase.
-        </li>
+      <section>
+        <h2>Generic DID URL Syntax</h2>
 
-        <li>
-The <a>DID method</a> name MUST be lowercase.
-        </li>
+        <p>
+          A <a>DID URL</a> always identifies a resource to
+          be located. It can be used, for example the specific part of a DID document.
+        </p>
+  
+        <p>
+  This following is the ABNF definition using the syntax in [[!RFC5324]]. It builds on the <code>did</code> scheme defined in <a href="#generic-did-syntax"></a>. The <code>path-abempty</code>, <code>query</code>, and
+  <code>fragment</code> components are identical to the ABNF rules defined in
+  [[!RFC3986]].
+        </p>
+  
+  
+        <pre class="nohighlight">
+  did-url            = did *( ";" param ) path-abempty [ "?" query ]
+                       [ "#" fragment ]
+  param              = param-name [ "=" param-value ]
+  param-name         = 1*param-char
+  param-value        = *param-char
+  param-char         = ALPHA / DIGIT / "." / "-" / "_" / ":" /
+                       pct-encoded
+        </pre>
+    
+      </section>
+  
+      <section>
+        <h2>Generic DID Parameters</h2>
```

Comment:
```suggestion
        <h2>Generic DID URL Parameters</h2>
```

Reply:
Done in https://github.com/w3c/did-core/pull/212/commits/9c6b3618db428edc90ebd42e1e28d77059560867